spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
[This rule](https://palantir.github.io/tslint/rules/no-unused-variable/) doesn't require type-checking.